lib/allowed-features.js

```diff
@@ -62,28 +76,19 @@ const hasFeature = (collective, feature) => {
     return false;
   }
 
-  // Allow Host Collectives to receive expenses
-  if (feature === FEATURES.RECEIVE_EXPENSES && collective.isHost) {
-    return true;
-  }
+  if (collective.features) {
+    const featureStatus = getFeatureStatus(collective.features, feature);
```

Comment:
I'm not convinced by this `getFeatureStatus` helper, it's actually more verbose than writing ddirecctly:

```suggestion
    const featureStatus = collective.features[feature];
```